### PR TITLE
[WIP] Website 2.0

### PR DESCRIPTION
**Pull request by sn3p**
- [x] Update deploy script (`.io` only for now)
- [x] Localized titles and meta tags
- [x] [Environment](https://middlemanapp.com/basics/upgrade-v4/#environments-and-changes-to-configure-blocks) per locale?
- [x] Update deploy script

### New website incoming :tada:

Photoshop mockup | Tour (first impression)
--- | ---
mockup **Alternative Quotes:** screen shot 2015-12-01 at 19 41 14 | tour

### CODE REVIEW

**Comment by sn3p** [comment]
#### Quote alternatives:

screen shot 2015-12-01 at 19 41 14

I personally like the big quote below best!

**Comment by sn3p** [comment]
**Tour, first impression:**

tour

**Comment by dkhgh** [comment]
wow great work

**Comment by tarzan** [comment]
very nice! :+1: 

Did you misspell my name on purpose? :japanese_goblin:

**Comment by sn3p** [comment]
Haha no, my bad :imp:

**Comment by fatboypunk** [comment]
:heart_eyes:

**Comment by sn3p** [comment]
### Website is now running on [Github Pages](http://defactosoftware.github.io/LearningSpaces-Landing/)
All paths generated by the `link_to` helper now prefix `/LearningSpaces-Landing` for build
```rb
configure :build do
  set :http_prefix, "/LearningSpaces-Landing"
end
```

This means from now on builds of v2 branch wil only work on gh-pages.
An alternative would be:
```rb
set :relative_links, true
activate :relative_assets
```
But somehow this didn't work for all paths.

**Comment by sn3p** [comment]
@RonaldDijkstra can you fix the `og` end `meta` tags? Would be awesome!

- [ ] Dynamic `meta` tags for `title` and `description`
- [ ] Dynamic `og` meta tags

File: source/localizable/_cta_signup.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<section class="cta-signup">
+  <div class="row">
+    <h3>Ready to give LearningSpaces a try?</h3>
```
Comment (fatboypunk) [inline source/localizable/_cta_signup.erb]: Do we `I18n` this?

Comment (sn3p) [inline source/localizable/_cta_signup.erb]: Yep, wip!

**Comment by sn3p** [comment]
I've [created](https://github.com/DefactoSoftware/LearningSpaces-Landing/blob/472d1f847095fb48b049b13312dad5dec54d60e5/config.rb#L37-L53) environments for the `en` and `nl` locales:

```bash
bundle exec middleman -e en
bundle exec middleman -e nl

bundle exec middleman build -e en
bundle exec middleman build -e nl
```

Still have to update the deploy script to include `nl`.

**Comment by fatboypunk** [comment]
> I've created environments for the en and nl locales:
> 
> bundle exec middleman -e en
> bundle exec middleman -e nl
> 
> bundle exec middleman build -e en
> bundle exec middleman build -e nl
> Still have to update the deploy script to include nl.
> Also note that serving or building single locales doesn't have i18n fallback.

Might be a good idea to add this to the readme

**Comment by sn3p** [comment]
Will do, but first wanted to ask you guys if this is best way to go.

**Comment by fatboypunk** [comment]
I think so, since we want to use our tld's and not `.io/nl`

**Pull request merged**